lib/maven_project.rb

```diff
@@ -58,7 +58,7 @@ def compiled?
 
   def compile!
     Dir.chdir(path) do
-      SystemCommands.sh!('mvn', '-q', 'package')
+      SystemCommands.sh!('mvn -q package -Dmaven.test.skip=true', {escape: false})
```

Comment:
Ok, strange that it affects the result (`echo \=` prints just `=`). Maybe just leave a comment about it then.